docs/source/user-guide/configuration/use-condarc.rst

```diff
@@ -141,6 +141,26 @@ Conda looks in the following locations for a ``.condarc`` file:
 ``CONDA_ROOT`` is the path for your base conda install.
 ``CONDA_PREFIX`` is the path to the current active environment.
 
+Conflict merging strategy
+-------------------------
+When conflicts between configurations arise, the following strategies are employed:
+
+* Lists - merge
+* Dictionaries - merge
+* Primitive - clobber
+
+Precedence
+----------
+
+The precedence by which the conda configuration is built out is shown below.
+Each new arrow takes precedence over the ones before it. For example, config
+files (by parse order) will be superceded by any of the other configuration
+options and environment variables will always take precedence over the other 3.
```

Comment:
I think it would be nice to mention how conda recognizes the config environment variables. They are of the form `CONDA_<name of config parameter>`